cmd/exchange_template/exchange_template.go

```diff
@@ -173,31 +172,38 @@ func makeExchange(exchangeDirectory string, configTestFile *config.Config, exch
 		{
 			Name:         "readme",
 			Filename:     "README.md",
-			TemplateFile: "readme_file.tmpl",
+			TemplateFile: "readme.tmpl",
 		},
 		{
-			Name:         "main",
-			Filename:     "main_file.tmpl",
-			FilePostfix:  ".go",
-			TemplateFile: "main_file.tmpl",
+			Name:         "rest",
+			Filename:     "rest.go",
+			TemplateFile: "main.tmpl",
 		},
 		{
 			Name:         "test",
 			Filename:     "test_file.tmpl",
 			FilePostfix:  "_test.go",
-			TemplateFile: "test_file.tmpl",
+			TemplateFile: "test.tmpl",
 		},
 		{
 			Name:         "type",
-			Filename:     "type_file.tmpl",
-			FilePostfix:  "_types.go",
-			TemplateFile: "type_file.tmpl",
+			Filename:     "types.go",
+			TemplateFile: "type.tmpl",
 		},
 		{
 			Name:         "wrapper",
-			Filename:     "wrapper_file.tmpl",
-			FilePostfix:  "_wrapper.go",
-			TemplateFile: "wrapper_file.tmpl",
+			Filename:     "wrapper.go",
+			TemplateFile: "wrapper.tmpl",
```

Comment:
🚧 🧵 [Outstanding thread](https://github.com/thrasher-corp/gocryptotrader/pull/1777#discussion_r2127806601); Please change to `gct.*`
If thrasher doesn't agree, he can comment when he reviews.